cmd/cloud-controller-manager/nodeipamcontroller.go

```diff
@@ -161,6 +161,26 @@ func startNodeIpamController(ccmConfig *cloudcontrollerconfig.CompletedConfig, n
 	return nil, true, nil
 }
 
+func validClusterCIDR(ccmConfig *cloudcontrollerconfig.CompletedConfig) ([]*net.IPNet, error) {
```

Comment:
this helper is only used in  cmd/cloud-controller-manager/gkenetworkparamsetcontroller.go , so it should be added in that file, avoiding a cross dependency on files that will complicate the code maintenance

Reply:
The helper function is used by the node ipam controller as well. I updated the PR.

If both controllers are using this helper function, should I add a `util.go` file under the same folder `cmd/cloud-controller-manager` as the controllers?